app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -65,12 +75,27 @@ fun ProfileScreen(
               horizontalArrangement = Arrangement.spacedBy(10.dp, Alignment.CenterHorizontally),
               verticalAlignment = Alignment.Top) {
 
-                // profile placeholder
-                Image(
-                    painter = painterResource(id = R.drawable.profile),
-                    contentDescription = "profile picture",
-                    modifier = Modifier.size(200.dp).testTag("profilePicture"),
-                )
+                // fetch profile picture from firebase
+                val auth = Firebase.auth
+                val user = auth.currentUser
+                val photo = user?.photoUrl
+                val placeholderUri =
+                    "https://picsum.photos/200/300".toUri() // Placeholder image URI
+
+                // display the profile picture
+                AsyncImage(
+                    model =
+                        ImageRequest.Builder(LocalContext.current)
+                            .data(photo ?: placeholderUri)
+                            .placeholder(R.drawable.profile)
+                            .build(),
+                    contentDescription = null,
```

Comment:
Please think about adding a `contentDescription`, it improve the app accesibility.

Reply:
added